go/internal/sidecar/sse_stream.go

```
@@ -0,0 +1,393 @@
+	"bytes"
+	"errors"
+	"math"
+	"strconv"
+	"sync/atomic"
+
+	"github.com/lidge-jun/opencodex/go/internal/jsonwire"
+)
+
+const maxResponsesSSEBlockBytes = 4 * 1024 * 1024
+
+var errResponsesSSEBlockTooLarge = errors.New("Responses SSE block exceeds maximum size")
```

golangci-lint (2.13.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)